### DISCUSSION TITLE
Google Maps design broken on emulator?

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#15542] [Q&A] [answered]
Can someone tell me, why Google maps looks kind of broken on all my emulators?
Happens on both unified and old map.

Selected theme is "Default" but its kind of the same on all themes:
Roads and some other elements are black on higher zoom levels. On low zoom it looks normal.

![image](https://github.com/cgeo/cgeo/assets/949669/cf2a821a-b8e6-4f5c-8aa4-7ee4251b6940)

![image](https://github.com/cgeo/cgeo/assets/949669/4ec56c10-7209-4a4e-80af-57f4d8a3ca22)

**Comment by SammysHP** [comment, marked as answer]
Appears to be a known problem on the emulator. You can find many similar reports:

https://www.google.com/search?q=google+maps+emulator+black+roads
